collector/reloads_config.go

```diff
@@ -0,0 +1,21 @@
+package collector
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type reloadsConfigCollector struct {
+	Failures  *prometheus.Desc
+	Successes *prometheus.Desc
+}
+
+func newReloadsConfigCollector() *reloadsConfigCollector {
+	desc := newDescFunc(namespace, "reloads_config")
+	return &reloadsConfigCollector{
+		Failures:  desc("failures", "Number of failures during config reload."),
+		Successes: desc("successes", "Number of successful config reloads."),
+	}
+}
+
+func (c *reloadsConfigCollector) Collect(p ReloadsConfig, ch chan<- prometheus.Metric) {
+	ch <- prometheus.MustNewConstMetric(c.Failures, prometheus.GaugeValue, float64(p.Failures))
```

Comment:
I think the type of `successes` and `failures` are counter.
How about changing it to prometheus.CounterValue?

Reply:
Nice catch, thank you!